packages/manager/modules/overthebox/src/overthebox/migration/offers/overTheBox-migration-offers.controller.js

```diff
@@ -0,0 +1,91 @@
+export default class OverTheBoxMigrationOffersCtrl {
+  /* @ngInject */
+  constructor($q, $scope, $translate, TucToast, OverTheBoxMigrationService) {
+    this.$q = $q;
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.TucToast = TucToast;
+    this.OverTheBoxMigrationService = OverTheBoxMigrationService;
+  }
+
+  $onInit() {
+    this.offers = [];
+    this.loading = true;
+
+    this.$q.all([this.getCurrentOffer(), this.getOffers()]).finally(() => {
```

Comment:
Do we catch errors here? Or what happen if an error occur?

Reply:
For the first call getCurrentOffer(), we shoudn't have any error because if we are into migration, the detail of the overthebox was ok before.
For the second call getOffers(), the error message is displayed into the view and no offer is displayed.